pkg/driver_cryptocell/psa_cc/Makefile.dep

```diff
@@ -0,0 +1,30 @@
+USEMODULE += psa_cc
+USEMODULE += psa_cc_error_conversion
+
+ifneq (,$(filter psa_cc_hashes_sha1,$(USEMODULE)))
+  USEMODULE += psa_cc_hashes_common
+endif
+
+ifneq (,$(filter psa_cc_hashes_sha224,$(USEMODULE)))
+  USEMODULE += psa_cc_hashes_common
+endif
+
+ifneq (,$(filter psa_cc_hashes_sha256,$(USEMODULE)))
+  USEMODULE += psa_cc_hashes_common
+endif
+
+ifneq (,$(filter psa_cc_hashes_sha512,$(USEMODULE)))
+  USEMODULE += psa_cc_hashes_common
```

Comment:
As discussed offline: makes sense to consider wildcards or do you need to be explicit?
```
ifneq (,$(filter psa_cc_hashes_*,$(USEMODULE)))
  USEMODULE += psa_cc_hashes_common
```

Reply:
Hmm at least when handling pseudomodules I was asked to be explicit, so I think it would be a good idea to keep that up.